stable/eventstore/templates/eventstore-statefulset.yaml

```diff
@@ -57,10 +57,12 @@ spec:
                 fieldRef:
                   fieldPath: status.podIP
             {{- end }}
+            {{- if and .Values.extIp (gt .Values.clusterSize 1.0) }}
```

Comment:
This new condition make use of EXT_IP_ADVERTISE_AS only if extIp is set. In case is not, the status.podIP is assigned to EXT_IP and IP_ADVERTISE_AS is not added, right? 
It looks ok to me

Reply:
Yes, `EXT_IP_ADVERTISE_AS` is only populated if the `EXT_IP` is defined and if the `CLUSTER_SIZE` is greater than 1.